j2lint/utils.py

```diff
@@ -25,17 +23,18 @@ def load_plugins(directory):
     """
     result = []
     file_handle = None
-
-    for pluginfile in glob.glob(os.path.join(directory, '[A-Za-z]*.py')):
+    for pluginfile in glob.glob(os.path.join(directory, '[A-Za-z_]*.py')):
         pluginname = os.path.basename(pluginfile.replace('.py', ''))
         try:
-            logger.debug("Loading plugin {}".format(pluginname))
+            logger.debug("Loading plugin %s", pluginname)
             spec = importlib.util.spec_from_file_location(
                 pluginname, pluginfile)
-            module = importlib.util.module_from_spec(spec)
-            spec.loader.exec_module(module)
-            obj = getattr(module, pluginname)()
-            result.append(obj)
+            if pluginname != "__init__":
+                class_name = ''.join(str(name).capitalize()for name in pluginname.split('_'))
```

Comment:
Hmm so here we are now assuming that if the user want to use a different directory of rules they will always name the class based on the filename and doing the same process (removing `_` and capitalizing)

Here the goal for us is not to refactor this either but if this is not documented it is quite hard to maintain this. We have a section in the README indicating how to add custom rules maybe worth mentioning it there.

Reply:
Added to the readme.